iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Authentication;
+using Microsoft.Azure.Devices.Extensions;
+using Microsoft.Azure.Devices.Generated;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// /// The Digital Twins Service Client contains methods to retrieve and update digital twin information, and invoke commands on a digital twin device.
+    /// </summary>
+    public class DigitalTwinClient : IDisposable
+    {
+        private readonly IotHubGatewayServiceAPIs _client;
```

Comment:
do we use this client anywhere else other than the constructor? if that's so we shouldn't make it a private field.

Reply:
Sorry I didn't catch that. It's only used in the constructor, and it's used to instantiate a new DigitalTwin object. Why not leave it as private?

Reply:
oh, I didn't mean it should be public, I meant we don't need to have a field for it at all, it can be a local variable in the constructor

Reply:
It turns out it's being disposed in Dispose. Leaving as is. Discussed with Azad offline.